DigitalLearningSolutions.Web/Views/LearningMenu/Index.cshtml

```diff
@@ -15,3 +15,12 @@
     <p class="nhsuk-u-secondary-text-color">@Model.CourseContent.BannerText</p>
   </div>
 </div>
+
+<div class="nhsuk-back-link">
+  <a class="nhsuk-back-link__link" asp-action="Current" asp-controller="LearningPortal">

```

Comment:
I think you're right and a back chevron might be better than the x. We might need to change it later (at least for some of the other pages) because we might want some different back buttons (e.g. to bring the user back to the previous section). But we can deal with that if and when we get to it 🙂